features/exacte_match.feature

```diff
@@ -0,0 +1,188 @@
+  | 2391PH   | 3          | a          |                      | 1672200000005237              |
+  | 2391PH   | 5          |            |                      | 1672200000005239              |
+  | 2391PH   | 7          |            |                      | 1672200000009548              |
+  | 2391PH   | 9          |            |                      | 1672200000008470              |
+  | 2391PH   | 11         |            |                      | 1672200000005233              |
+  | 2594AN   | 265        |            |                      | 0518200000390367              |
+  | 2594AN   | 265        | A          |                      | 0518200000452568              |
+  | 2594AN   | 265        | A          | 1                    | 0518200000452570              |
+  | 2594AN   | 265        | A          | 2                    | 0518200000620759              |
+  | 2594AN   | 265        | A          | 3                    | 0518200000625989              |
+  | 2594AN   | 265        | B          |                      | 0518200000454095              |
+  | 2594AN   | 265        | B          | 1                    | 0518200001647713              |
+  | 2594AN   | 265        | B          | 2                    | 0518200001647715              |
+  | 2594AN   | 265        | B          | 3                    | 0518200001647717              |
+  | 2594AN   | 265        | B          | 3                    | 0518200001775151              |
+  | 2594AN   | 265        | C          |                      | 0518200001664062              |
+  | 2594AN   | 265        | C          | 1                    | 0518200001664064              |
+  | 2594AN   | 265        | C          | 2                    | 0518200001664066              |
+  | 2594AN   | 265        | C          | 3                    | 0518200001664068              |
+  | 2594AN   | 265        | D          |                      | 0518200000411785              |
+  | 2594AN   | 265        | E          |                      | 0518200001639143              |
+  | 2594AN   | 265        | F          |                      | 0518200001639145              |
+  | 7411CC   | 36         |            |                      | 0150200000052413              |
+  | 7411CC   | 36         |            |                      | 0150200000039067              |
+  | 7437SZ   | 2          | A          |                      | 0150200000000313              |
+  | 7437SZ   | 2          | A          |                      | 0150200000052567              |
+  | 1015DT   | 237        | A          | V                    | 0363200000459979              |
+  | 1015DT   | 237        | C          | A                    | 0363200000459980              |
+  | 3581KL   | 49         |            | A                    | 0344200000172058              |
+
+Scenario: Zoeken met combinatie postcode en huisnummer
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
```

Comment:
Het is niet nodig om deze Gegeven op te nemen. Met een Achtergrond geeft je aan dat alle Gegeven-s onder de achtergrond voor elke scenario in de feature file geldt